client/packages/inventory/src/Stocktake/DetailView/ContentArea/columns.ts

```diff
@@ -56,6 +56,26 @@ export const useStocktakeColumns = ({
 
   return useColumns<StocktakeLineFragment | StocktakeSummaryItem>(
     [
+      {
+        key: 'locationCode',
+        label: 'label.location',
+        getSortValue: row => {
+          if ('lines' in row) {
+            const { lines } = row;
+            return (
+              lines.reduce((_, line) => line.location?.code ?? '', '') ?? ''
+            );
+          }
+          return row.location?.code ?? '';
+        },
+        accessor: ({ rowData }) => {
+          if ('lines' in rowData) {
+            const { lines } = rowData;
+            return lines.reduce((_, line) => line.location?.code ?? '', '');
+          }
+          return rowData.location?.code;
+        },
+      },
```

Comment:
I think location name is more useful? In which case you can use the existing column definition and just extend to get the sort value and accessor:

```suggestion
[
        'locationName',
        {
          getSortValue: row => {
            if ('lines' in row) {
              const { lines } = row;
              return (
                lines.reduce((_, line) => line.location?.name ?? '', '') ?? ''
              );
            }
            return row.location?.code ?? '';
          },
          accessor: ({ rowData }) => {
            if ('lines' in rowData) {
              const { lines } = rowData;
              return lines.reduce((_, line) => line.location?.name ?? '', '');
            }
            return rowData.location?.code;
          },
        },
      ],
```

Reply:
Had tried extending column for location code but was getting type error. So used key, label, ... way define location code column. Though it was working with location name without any errors. Have change location code to name. f7ea6f2ff